components/ui/BlogPagination.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from 'react'
+import styled, { css } from 'styled-components'
+import Link from 'next/link'
+import Router from 'next/router'
+
+import { DynamicLink } from './DynamicLink'
+
+export const BlogPagination = styled(
+  ({ currentPage, numPages, ...styleProps }) => {
+    const isFirst = currentPage === 1
+    const isLast = currentPage === numPages
+    const prevPage =
+      currentPage - 1 === 1 ? '/blog' : `/blog/page/${currentPage - 1}`
+    const nextPage = `/blog/page/${currentPage + 1}`
+    const [selectValue, setSelectValue] = useState(currentPage)
+
+    function handleSelectChange(e) {
+      e.preventDefault()
+      const pageNumber = e.target.value
+      setSelectValue(pageNumber)
+      if (pageNumber === '1') {
```

Comment:
I don't think we need this conditional, as /blog/page/1 should work

Reply:
Tested and indeed that works, but I do like that this keeps the URL clean (/blog vs. /blog/page/1). I'm fine with removing it, though. Simplifies things.

Reply:
@kendallstrautman What do you think?

Reply:
yeah that's fine as long as `/blog` still works

Reply:
ill fix this in another branch